content/en/serverless/guide/serverless_tracing_and_bundlers.md

```diff
@@ -126,6 +126,78 @@ Datadog's tracing libraries (`dd-trace`) are known to be not compatible with bun
     module.exports = [nodeExternalsPlugin()]
     ```
 
+## AWS CDK & esbuild
+
+The `NodeJsFunction` construct in the AWS CDK uses esbuild. The default configuration is not compatible with Datadog's tracing libraries. The CDK allows you to override the default configuration and provide a custom esbuild file to support bundling and the Datadog tracing libraries:
+
+1. Follow the installation instructions for Node.js and ensure the Datadog Lambda layer for Node.js is added to your Lambda function.
+2. Remove datadog-lambda-js and dd-trace from your package.json and the build process, since they are already available in the Lambda runtime provided by the Datadog Lambda layer.
```

Comment:
```suggestion
2. Remove `datadog-lambda-js` and `dd-trace` from your package.json and the build process, since they are already available in the Lambda runtime provided by the Datadog Lambda Layer.
```